functions.sh

```diff
@@ -710,6 +710,23 @@ function pick_editor()
     # editor="BASIC"
 }
 
+function pick_theme()
+{
+    theme=$(dialog --backtitle "$banner"\
+        --title "...Colors..."\
+        --no-cancel\
+        --menu "What style do you prefer?"\
+        10 50 3\
+        "default"    "The classic Analog City look"\
+        "nighttime"  "Dark mode, Analog City at night"\
+        "monochrome" "Like a Macintosh I"\
+        3>&1 1>&2 2>&3 3>&-\
+        )
+
+    if [ -z "$theme" ]; then theme="default"; fi
+    export DIALOGRC="$(pwd)/themes/$theme.dialogrc"
```

Comment:
My only concern with this PR is that the "$(pwd)" bit might not work on your server, depending on what directory `shell.sh` gets invoked from.

This might need to change based on the value of the `debug` variable.